### PR TITLE
Replace global with DIC globals in ilCronManager

### PR DESCRIPTION
**Pull request by legionth**
Refers to https://www.ilias.de/mantis/view.php?id=22869

### CODE REVIEW

**Comment by legionth** [comment]
Ping @mjansenDatabay

File: Services/Cron/classes/class.ilCronManager.php [hunk 1]
```diff
@@ -120,8 +122,11 @@ public static function runJobManual($a_job_id)
 	 */
 	protected static function runJob(ilCronJob $a_job, array $a_job_data = null, $a_manual = false)
 	{
-		global $ilLog, $ilDB;
-		
+		global $DIC;
+
+		$ilLog = $DIC->logger();
```
Comment (mjansenDatabay) [inline Services/Cron/classes/class.ilCronManager.php]: `$DIC->logger()->root()`

File: Services/Cron/classes/class.ilCronManager.php [hunk 2]
```diff
@@ -239,8 +244,11 @@ protected static function runJob(ilCronJob $a_job, array $a_job_data = null, $a_
 	 */
 	public static function getJobInstanceById($a_job_id)
 	{
-		global $ilLog, $ilPluginAdmin;
-				
+		global $DIC;
+
+		$ilLog = $DIC->logger();
```
Comment (mjansenDatabay) [inline Services/Cron/classes/class.ilCronManager.php]: `$DIC->logger()->root()`

File: Services/Cron/classes/class.ilCronManager.php [hunk 3]
```diff
@@ -290,8 +298,10 @@ public static function getJobInstanceById($a_job_id)
 	 */
 	public static function getJobInstance($a_id, $a_component, $a_class, $a_path = null)
 	{
-		global $ilLog;
-		
+		global $DIC;
+
+		$ilLog = $DIC->logger();
```
Comment (mjansenDatabay) [inline Services/Cron/classes/class.ilCronManager.php]: `$DIC->logger()->root()`

File: Services/Cron/classes/class.ilCronManager.php [hunk 4]
```diff
@@ -346,8 +356,13 @@ protected static function sendNotification(ilCronJob $a_job, $a_message)
 	
 	public static function createDefaultEntry(ilCronJob $a_job, $a_component, $a_class, $a_path)
 	{
-		global $ilDB, $ilLog, $ilSetting;
-		
+
+		global $DIC;
+
+		$ilLog = $DIC->logger();
```
Comment (mjansenDatabay) [inline Services/Cron/classes/class.ilCronManager.php]: `$DIC->logger()->root()`

File: Services/Cron/classes/class.ilCronManager.php [hunk 5]
```diff
@@ -439,8 +456,11 @@ public static function updateFromXML($a_component, $a_id, $a_class, $a_path = nu
 	 */
 	public static function clearFromXML($a_component, array $a_xml_job_ids)
 	{
-		global $ilDB, $ilLog;	
-		
+		global $DIC;
+
+		$ilDB = $DIC->database();
+		$ilLog = $DIC->logger();
```
Comment (mjansenDatabay) [inline Services/Cron/classes/class.ilCronManager.php]: `$DIC->logger()->root()`

File: Services/Cron/classes/class.ilCronManager.php [hunk 6]
```diff
@@ -82,8 +82,10 @@ public function runActiveJobs()
 	 */
 	public static function runJobManual($a_job_id)
 	{
-		global $ilLog;
-		
+		global $DIC;
+
+		$ilLog = $DIC->logger();
```
Comment (mjansenDatabay) [inline Services/Cron/classes/class.ilCronManager.php]: `$DIC->logger()->root()`

**Review by mjansenDatabay — CHANGES_REQUESTED** [review]

**Comment by legionth** [comment]
The code uses `$DIC->logger()->root()` now.

**Review by mjansenDatabay — APPROVED** [review]

**Pull request merged**